easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -453,6 +458,17 @@ def configure_step(self):
             except IOError:
                 raise EasyBuildError("Creating %s failed", self.sitecfgfn)
 
+        # conservatively auto-enable checking of $LDSHARED if it is not explicitely enabled or disabled
+        # only do this for sufficiently recent Python versions (>= 3.7 or Python 2.x >= 2.7.15)
+        if self.cfg.get('check_ldshared') is None:
+            pyver = det_python_version(self.python_cmd)
+            recent_py2 = pyver.startswith('2') and LooseVersion(pyver) >= LooseVersion('2.7.15')
+            if recent_py2 or LooseVersion(pyver) >= LooseVersion('3.7'):
```

Comment:
Do we need to go this recent?

Reply:
It's a conservative choice.

We're not actually building `Python` with `GCCcore` anyway (in the central repos) with older versions.

Reply:
Are you possibly using rpath?